FirstApp/Login/View/ViewController.swift

```diff
@@ -8,15 +8,30 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         setUpElements()
+       let gradientLayer = CAGradientLayer()
+           var updatedFrame = self.navigationController!.navigationBar.bounds
+           updatedFrame.size.height += self.navigationController!.navigationBar.frame.height
+           gradientLayer.frame = updatedFrame
+           gradientLayer.colors = [UIColor(red: 75/255,
+                                           green: 29/255,
+                                           blue: 87/255, alpha: 1).cgColor, UIColor(red: 152/255,
+                                                                                    green: 103/255,
+                                                                                    blue: 163/255, alpha: 1).cgColor]
+           gradientLayer.startPoint = CGPoint(x: 0.5, y: 0.0) // vertical gradient start
+           gradientLayer.endPoint = CGPoint(x: 0.5, y: 1.0) // vertical gradient end
+           UIGraphicsBeginImageContext(gradientLayer.bounds.size)
+           gradientLayer.render(in: UIGraphicsGetCurrentContext()!)
```

Comment:
Pretty sure this could be put in an extension if you use it elsewhere.

Reply:
Thanks, I don't but I'll keep this in mind if I do